platform/disk/sfdisk_partitioner_test.go

```diff
@@ -136,6 +145,20 @@ var _ = Describe("sfdiskPartitioner", func() {
 		Expect(len(runner.RunCommandsWithInput)).To(Equal(0))
 	})
 
+	It("sfdisk partition when partitions already match for mutlitpath", func() {
```

Comment:
Sorry, I submitted another [PR 94](https://github.com/cloudfoundry/bosh-agent/pull/94) to correct it.